src/main/java/seedu/address/logic/commands/MakeStatsCommand.java

```diff
@@ -1,22 +1,57 @@
 package seedu.address.logic.commands;
 
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.chart.PieChart;
+import seedu.address.commons.core.index.Index;
 import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.person.Person;
 
 /**
- * Format full help instructions for every command for display.
- */
+* Generates statistics for the event specified
+*/
 public class MakeStatsCommand extends Command {
 
     public static final String COMMAND_WORD = "makeStats";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Generates the Statistics of the Event at the user"
+        + "specified index. Type of statistics shown is based off a user input parameter.\n"
+        + "Parameters INDEX " + PREFIX_TYPE + "TYPE. \n"
+        + "Example: " + COMMAND_WORD + " 1 " + PREFIX_TYPE + "g";
 
     public static final String SHOWING_STATS_MESSAGE = "Opened statistics window.";
 
+    public final Index index;
+    public final boolean isGenderStatistic;
+
+    /**
+     * Constructs a {@code MakeStatsCommand} to create statistics
+     *  for the event at the specified {@code index}
```

Comment:
Had an extra space; Also, should we specify that its the index of the events list to make it clearer?
```suggestion
     * Constructs a {@code MakeStatsCommand} to create statistics
     * for the event at the specified {@code index} of the events list
```

Reply:
mmmm gd point will specify it is of the events list